felix/bpf/maps.go

```diff
@@ -47,6 +47,11 @@ type Map interface {
 	// Path returns the path that the map is (to be) pinned to.
 	Path() string
 
+	// GetMaxEntries returns the size of the map
+	GetMaxEntries() int
```

Comment:
It is common that Getters are without Get. There is no need for getter if the member is already exported.